app/config/storage/logos.php

```diff
@@ -2,6 +2,7 @@
 
 return [ // Based on this list @see http://stackoverflow.com/a/4212908/2299554
     'default' => __DIR__.'/logos/none.png',
+    'default_image' => __DIR__.'/logos/image.png',
```

Comment:
Can we stick a little to the Mime type naming and do something like `image/*`? 🤔

Reply:
Why, this is not the icon based on mime type, but rather a default icon for images larger than 10 mb, for image mimes we show preview?